tools/wptserve/wptserve/ranges.py

```diff
@@ -3,6 +3,7 @@
 
 class RangeParser(object):
     def __call__(self, header, file_size):
+        header = header.decode("ascii")
```

Comment:
Let's put this in a try block, with `except UnicodeDecodeError: raise HTTPException(400, "Non-ASCII range header value")`, rather than returning a somewhat mysterious error.